source/features/clone-branch.tsx

```diff
@@ -62,20 +62,20 @@ async function cloneBranch(event: delegate.Event<MouseEvent, HTMLButtonElement>)
 	}
 
 	textFieldEdit.set(
-		select<HTMLInputElement>('.js-branch-search-field')!,
+		select<HTMLInputElement>('[aria-label="Search branches…"]')!,
 		newBranchName
 	);
 }
 
 function init(): void | false {
-	const deleteIcons = select.all('[aria-label="Delete this branch"]');
-	// Is the user does not have rights to create a branch
+	const deleteIcons = select.all('.Details-content--shown .octicon.octicon-trashcan');
```

Comment:
This lets us skip the `.parentElement` below

```suggestion
	const deletionForms = select.all('form[data-action$="#destroy"]');
```

Reply:
I did that since 
![image](https://user-images.githubusercontent.com/16872793/79127172-68202380-7d6f-11ea-9287-55abc6a6dda1.png)

Reply:
Found another way